dronegen/windows.go

```diff
@@ -43,7 +42,7 @@ func windowsTagPipeline() pipeline {
 
 	p.DependsOn = []string{tagCleanupPipelineName}
 	p.Trigger = triggerTag
-
+	p.Concurrency.Limit = 1
```

Comment:
wasn';t it suppose to remove limit @tcsc ?

Reply:
> The concurrent limits have been left for tag builds, as there are flow-on concurrency concern due to artifact uploading, etc.